From 88aec6df5236f77caf0fa8520cfa2561e5ffb4dc Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Sun, 30 Dec 2018 19:44:02 -0600
Subject: [PATCH 2/2] hack: gpiolib: yes we have drivers stomping on each
 other, we need to find a better way to share gpio...

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 drivers/gpio/gpiolib.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpio/gpiolib.c b/drivers/gpio/gpiolib.c
index b02cc2abd3b6..402d1efd8b18 100644
--- a/drivers/gpio/gpiolib.c
+++ b/drivers/gpio/gpiolib.c
@@ -1836,9 +1836,9 @@ static int gpiod_request_commit(struct gpio_desc *desc, const char *label)
 
 	if (test_and_set_bit(FLAG_REQUESTED, &desc->flags) == 0) {
 		desc_set_label(desc, label ? : "?");
-	} else {
-		ret = -EBUSY;
-		goto out_free_unlock;
+	//} else {
+		//ret = -EBUSY;
+		//goto out_free_unlock;
 	}
 
 	if (gc->request) {
-- 
2.29.2

